### ISSUE TITLE
All tests fail right now

### ISSUE THREAD

**Issue by polluks** [gwsw/less#305]
```
FAIL chinese1 on cmd #0 (. 0)
FAIL chinese1 (0 steps)
ERR  status 256 from /home/stefan/g/less/lesstest/lesstest  -e -s '/home/stefan/g/less/lesstest/lt_screen' -t '/home/stefan/g/less/lesstest/lt/chinese1.lt' '/home/stefan/g/less/less'
FAIL colorbars on cmd #0 (. 0)
FAIL colorbars (0 steps)
ERR  status 256 from /home/stefan/g/less/lesstest/lesstest  -e -s '/home/stefan/g/less/lesstest/lt_screen' -t '/home/stefan/g/less/lesstest/lt/colorbars.lt' '/home/stefan/g/less/less'
FAIL github216 on cmd #0 (. 0)
FAIL github216 (0 steps)
ERR  status 256 from /home/stefan/g/less/lesstest/lesstest  -e -s '/home/stefan/g/less/lesstest/lt_screen' -t '/home/stefan/g/less/lesstest/lt/github216.lt' '/home/stefan/g/less/less'
FAIL github265 on cmd #0 (. 0)
FAIL github265 (0 steps)
ERR  status 256 from /home/stefan/g/less/lesstest/lesstest  -e -s '/home/stefan/g/less/lesstest/lt_screen' -t '/home/stefan/g/less/lesstest/lt/github265.lt' '/home/stefan/g/less/less'
FAIL seq200 on cmd #0 (. 0)
FAIL seq200 (0 steps)
ERR  status 256 from /home/stefan/g/less/lesstest/lesstest  -e -s '/home/stefan/g/less/lesstest/lt_screen' -t '/home/stefan/g/less/lesstest/lt/seq200.lt' '/home/stefan/g/less/less'
FAIL utf8-2.txt on cmd #0 (. 0)
FAIL utf8-2.txt (0 steps)
ERR  status 256 from /home/stefan/g/less/lesstest/lesstest  -e -s '/home/stefan/g/less/lesstest/lt_screen' -t '/home/stefan/g/less/lesstest/lt/utf8-2.txt.lt' '/home/stefan/g/less/less'
FAIL version.c on cmd #0 (. 0)
FAIL version.c (0 steps)
ERR  status 256 from /home/stefan/g/less/lesstest/lesstest  -e -s '/home/stefan/g/less/lesstest/lt_screen' -t '/home/stefan/g/less/lesstest/lt/version.c.lt' '/home/stefan/g/less/less'
RAN  7 tests with 7 errors

**Comment by gwsw** [comment]
Should be fixed in 7b4da16e59e544815ea0e50b46c34d7c5acd9a8c (less-612).

**Comment by polluks** [comment]
thx
